lib/vagrant-mutate/mutate.rb

```diff
@@ -41,7 +46,12 @@ def execute
       output_loader = BoxLoader.new(@env)
       output_box = output_loader.prepare_for_output(input_box.name, options[:output_provider], input_box.version)
 
-      converter  = Converter::Converter.create(@env, input_box, output_box)
+      if options[:force_virtio] == true 
+        converter  = Converter::Converter.create(@env, input_box, output_box, options[:force_virtio])
+      else
+        converter  = Converter::Converter.create(@env, input_box, output_box)
+      end
+
```

Comment:
Instead of needing to call `create` two different ways, you could set `options[:force_virtio] = false` before you check for that option with `o.on` above, then always pass it through to `create`.